frontend/lib/src/components/widgets/AudioInput/AudioInput.tsx

```diff
@@ -203,16 +203,24 @@ const AudioInput: React.FC<Props> = ({
   )
 
   const handleClear = useCallback(
-    ({ updateWidgetManager }: { updateWidgetManager?: boolean }) => {
+    ({
+      updateWidgetManager,
+      deleteFile,
+    }: {
+      updateWidgetManager: boolean
+      deleteFile: boolean
+    }) => {
       if (isNullOrUndefined(wavesurfer) || isNullOrUndefined(deleteFileUrl)) {
         return
       }
       setRecordingUrl(null)
       wavesurfer.empty()
-      uploadClient.deleteFile(deleteFileUrl)
+      if (deleteFile) {
+        uploadClient.deleteFile(deleteFileUrl)
+      }
+      setDeleteFileUrl(null)
       setProgressTime(STARTING_TIME_STRING)
       setRecordingTime(STARTING_TIME_STRING)
-      setDeleteFileUrl(null)
```

Comment:
not sure what difference this makes, but I think we could still delete the delete file url since I think won't be needed on the frontend. After running this clear, the frontend won't have any handle on the file anyways. It only exists in the backend.

Reply:
oh i just moved it up to make it closer to the other delete, but it isn't behind the if statement, otherwise you get the weird state of a blank ui with the ability to clear/delete

Reply:
Oh, I see.. sorry I didn't saw that it was just moved up :)